content/applications/hr/employees/new_employee.rst

```diff
@@ -0,0 +1,269 @@
+- :guilabel:`Company`: Select the company from the drop-down menu that the new employee is hired by,
+  or create a new company by typing the name in the field.
+- :guilabel:`Working Hours`: In the :guilabel:`Work Information` tab, select the desired
+  :guilabel:`Working Hours` from the drop-down menu.
+
+.. image:: new_employee/working-hours.png
+   :align: center
+   :alt: Working Hours are located in the Work Information tab.
+
+.. note::
+   :guilabel:`Working Hours` are related to a company's working times, and an employee cannot have
+   working hours that are outside of a company's working times.
+
+   Each individual working time is company-specific, so for multi-company databases, each company
+   needs to have its own working hours set.
+
+   If an employee's working hours are not configured as a working time for the company, new working
+   times can be added, or existing working times can be modified. To add or modify a working time,
+   go to the :menuselection:`Payroll app --> Configuration --> Working Times`, and add a new working
+   time or edit an existing one.
+
+   After the new working time is created, set the working hours for the employee.
+
+Optional fields
+---------------
+
+- :guilabel:`Photo`: In the top right image box of the employee card, click on the :guilabel:`✏️
+  (pencil)` edit icon to select a photo to upload.
+- :guilabel:`Job Position`: Enter the employee's job position title.
+- :guilabel:`Tags`: Click on a tag in the drop-down menu to add any tags applicable to the employee.
+  Any tag can be created in this field by typing it in. Once created, the new tag is available for
+  all employee cards. There is no limit to the amount of tags that can be added.
+- Work Contact Information: Enter the employees :guilabel:`Work Mobile`, :guilabel:`Work Phone`,
+  :guilabel:`Work Email`, and/or :guilabel:`Company` name. The :guilabel:`External Link` button
+  (next to :guilabel:`Company`) opens the company's :guilabel:`General Information` card.
+  Modifications can be made to this card. Click :guilabel:`Save` after making edits.
+- :guilabel:`Department`: Select the employee's department from the drop-down menu. The
+  :guilabel:`External Link` button opens the selected department's details. Modifications to the
+  :guilabel:`Department Name`, :guilabel:`Parent Department`, :guilabel:`Manager`, and
+  :guilabel:`Company` can be made here. Click :guilabel:`Save` after making edits.
+- :guilabel:`Manager`: Select the employee's manager from the drop-down menu. The
+  :guilabel:`External Link` button opens the selected manager's employee form. Modifications to the
+  form can be made here. Click :guilabel:`Save` after any edits are made.
+- :guilabel:`Coach`: Select the employee's coach from the drop-down menu. The :guilabel:`External
+  Link` button opens the selected coach's employee form. Modifications to the form can be made here.
+  Click :guilabel:`Save` after any edits are made.
```

Comment:
The same rolling supplemental information on these three list items (below) can probably be omitted from the list items and turned into a single **tip** block.
```
  The :guilabel:`External
  Link` button opens the selected coach's employee form. Modifications to the form can be made here.
  Click :guilabel:`Save` after any edits are made.
```

Reply:
Great idea- and makes the list shorter/less repetitive!